notebooks/SparkPolaris.ipynb

```diff
@@ -258,7 +258,7 @@
     "\n",
     "spark = (SparkSession.builder\n",
     "  .config(\"spark.sql.catalog.spark_catalog\", \"org.apache.iceberg.spark.SparkSessionCatalog\")\n",
-    "  .config(\"spark.jars.packages\", \"org.apache.iceberg:iceberg-spark-runtime-3.5_2.13:1.5.0,org.apache.hadoop:hadoop-aws:3.4.0,software.amazon.awssdk:bundle:2.23.19,software.amazon.awssdk:url-connection-client:2.23.19\")\n",
+    "  .config(\"spark.jars.packages\", \"org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:1.5.0,org.apache.hadoop:hadoop-aws:3.4.0,software.amazon.awssdk:bundle:2.23.19,software.amazon.awssdk:url-connection-client:2.23.19\")\n",
```

Comment:
Not a blocker: we may update the iceberg version to 1.5.2 as well. 

Here is another place to upgrade as well:
https://github.com/polaris-catalog/polaris/blob/68b248aa98a527fe2f15a8920d7524adf288d6ed/regtests/t_pyspark/src/iceberg_spark.py#L75-L75

Reply:
Goot catch. Let me update that as well shortly.